### PR TITLE
Feature/result set

### PR DESCRIPTION
**Pull request by DarkSide666**

### CODE REVIEW

File: src/Expression.php [hunk 1]
```diff
@@ -111,6 +109,27 @@ public function offsetGet($offset) {
     }
 
     /**
+     * Use this instead of "new Expression()" if you want to automatically bind
+     * expression to the same connection as the parent.
+     */
+    public function expr($expr, $options = [])
+    {
+        $options['connection'] = $this->connection;
+        $class = get_class($this);
```
Comment (DarkSide666) [inline src/Expression.php]: Unused variable. Probably in next line should be something like `new $class(...)`

File: src/Expression.php [hunk 2]
```diff
@@ -283,4 +302,67 @@ public function getDebugQuery()
...
+                if (!$statement->bindValue($key,$val,$type)) {
+                    throw new Exception('Unable to bind parameter');
+                }
+            }
+
+            $statement->setFetchMode(\PDO::FETCH_ASSOC);
+            $statement->execute();
+            return $statement;
+            
+        } else {
```
Comment (DarkSide666) [inline src/Expression.php]: `else` statement here is not needed

File: src/Expression.php (hunk 1 above)
Comment (romaninsh) [inline src/Expression.php]: I was thinking about duplicating same class, but not sure how to approach this properly

Comment (DarkSide666) [inline src/Expression.php]: I guess `return new Expression` is OK. At least IDEs and analyzers will like that better :)

**Pull request merged**